client/components/bcaap-form-sections/funding-eligibility.tsx

```diff
@@ -134,19 +134,19 @@ export const FundingEligibility: React.FC = () => {
             <div className='flex flex-1 w-full justify-start'>
               <Radio
                 legend='The percentage share of BCAAP funding available for your project may be increased by an additional 5% if you are a non-profit society registered in British Columbia for the purposes of operating this small facility and you are reliant on volunteer contributions to complete this project. Does this description apply to you?'
```

Comment:
Maybe update legend here to remove BCAAP?
Also, the folder is called `bcaap-from-sections`.